expense/expenseController.js

```diff
@@ -0,0 +1,86 @@
+const router = require('express').Router();
+const Expense = require('./expenseModel.js');
+
+// /api/expense
+router
+  .route('/')
+  .get((req, res) => {
+    Expense.find({})
+    .populate('budget category')
```

Comment:
We can add a second argument to `populate()` and tack on `select()` at the end to remove the unnecessary info from our query.

```.populate('budget category', 'budgetAmount title -_id').select('-_id -__v')```